server/src/main/java/org/apache/iotdb/db/query/executor/fill/ValueFill.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.query.executor.fill;
+
+import org.apache.iotdb.db.metadata.PartialPath;
+import org.apache.iotdb.db.query.context.QueryContext;
+import org.apache.iotdb.tsfile.exception.write.UnSupportedDataTypeException;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.read.TimeValuePair;
+import org.apache.iotdb.tsfile.read.filter.basic.Filter;
+import org.apache.iotdb.tsfile.utils.Binary;
+import org.apache.iotdb.tsfile.utils.TsPrimitiveType;
+
+import java.util.Set;
+
+public class ValueFill extends IFill {
+  private PartialPath seriesPath;
+  private QueryContext context;
+  private String value;
+  private Set<String> allSensors;
+  private Filter timeFilter;
+
+  public ValueFill(TSDataType dataType, long queryTime, String value) {
+    super(dataType, queryTime);
+    this.value = value;
+  }
+
+  public ValueFill(String value) {
+    this.value = value;
+  }
+
+  @Override
+  public IFill copy() {
+    return new ValueFill(dataType, queryTime, value);
+  }
+
+  @Override
+  public void configureFill(
+      PartialPath path,
+      TSDataType dataType,
+      long queryTime,
+      Set<String> deviceMeasurements,
+      QueryContext context) {
+    this.seriesPath = path;
+    this.dataType = dataType;
+    this.context = context;
+    this.queryTime = queryTime;
+    this.allSensors = deviceMeasurements;
+  }
+
+  @Override
+  public TimeValuePair getFillResult() {
+    switch (dataType) {
+      case BOOLEAN:
+        return new TimeValuePair(
```

Comment:
We could cache the primitive value in this class to avoid parse each time, like this:

boolean booleanValue;
int intValue；
long longValue;
float floatValue;
double doubleValue;

Reply:
The initialization of value is placed in the constructor，configureFill is null , avoid parse each time